### PR TITLE
add option to not blank out diagonal in plot_mod_heatmap

### PR DESCRIPTION
**Pull request by koudyk**

### CODE REVIEW

File: netneurotools/plotting.py [hunk 1]
```diff
@@ -74,8 +74,7 @@ def sort_communities(consensus, communities):
 
 def plot_mod_heatmap(data, communities, *, inds=None, edgecolor='black',
                      ax=None, figsize=(6.4, 4.8), xlabels=None, ylabels=None,
-                     xlabelrotation=90, ylabelrotation=0, cbar=True,
-                     **kwargs):
+                     xlabelrotation=90, ylabelrotation=0, cbar=True, blank_diagonal=True, **kwargs):
```
Comment (rmarkello) [inline netneurotools/plotting.py]:
```suggestion
                     xlabelrotation=90, ylabelrotation=0, cbar=True, 
                     blank_diagonal=True, **kwargs):
```

File: netneurotools/plotting.py [hunk 2]
```diff
@@ -121,7 +122,11 @@ def plot_mod_heatmap(data, communities, *, inds=None, edgecolor='black',
         fig, ax = plt.subplots(1, 1, figsize=figsize)
 
     # plot data re-ordered based on community and node strength
-    plot_data = np.ma.masked_where(np.eye(len(data)), data[np.ix_(inds, inds)])
+    if blank_diagonal:
+        plot_data = np.ma.masked_where(np.eye(len(data)), data[np.ix_(inds, inds)])
```
Comment (rmarkello) [inline netneurotools/plotting.py]:
```suggestion
        plot_data = np.ma.masked_where(np.eye(len(data)), 
                                       data[np.ix_(inds, inds)])
```

File: netneurotools/plotting.py [hunk 3]
```diff
@@ -104,6 +103,8 @@ def plot_mod_heatmap(data, communities, *, inds=None, edgecolor='black',
         provided. Default : xlabelrotation: 90, ylabelrotation: 0
     cbar : bool, optional
         Whether to plot colorbar. Default: True
+    blank_diagonal : bool, optional
+        Whether to blank out the diagnonal. Default: True
```
Comment (rmarkello) [inline netneurotools/plotting.py]:
```suggestion
        Whether to mask the diagonal in the plotted heatmap. Default: True
```

**Comment by koudyk** [comment]
Thanks @rmarkello! I agree about the parameter name, mask_diagonal is better. Should I change it on my end and then do another pull request?

**Pull request merged**